api/api.go

```diff
@@ -1131,7 +1130,7 @@ func (s *SdkStoragePolicy) IsPermitted(ctx context.Context, accessType Ownership
 }
 
 func (s *SdkStoragePolicy) IsPermittedFromUserInfo(user *auth.UserInfo, accessType Ownership_AccessType) bool {
-	if s.IsPublic() {
+	if s.IsPublic(Ownership_Read) {
```

Comment:
I chose to still pass in Ownership_Read for the Public checks, since Public is always Ownership_Read, right?

Reply:
Updated all of these to pass in accessType instead.